products/grafana.md

```diff
@@ -20,8 +20,8 @@ releases:
     eol: false
     support: true
     releaseDate: 2022-11-29
-    latest: "9.3.2"
-    latestReleaseDate: 2022-12-14
+    latest: "9.3.4"
```

Comment:
LGTM : tags 9.3.3 does not exist (https://github.com/grafana/grafana/releases/tag/v9.3.3).